### ISSUE TITLE
A11y_SQLdatabase projects extension_Publish_Generate script_Status Messages: Voiceover is not announcing the "Error Message" getting while generating script.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/azuredatastudio#22163]
### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #WCAG4.1.3; #MAC; #AINotInScope; #Status Messages; #Voiceover; #A11yWCAG2.1;

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

### Repro Steps:

1. Start Voiceover with using shortcut key Command+F5.
2. Launch Azure Data Studio
3. Tab till extension and search for “SQL Database Projects” and install it.
4. Tab till database project and press enter.
5. “Database Project” panel will open.
6. Tab till existing project name and press context menu button(right click).
7. Tab till "Publish" menu-item in list and press enter.
8. Fill all the required details and connect to the database.
9. Tab till "Generate Script" button and press enter.
10. "Build Error" message will appear while the output window is getting updated.
11. Verify whether Voiceover is announcing the "Error Message" getting while generating script or not.

### Actual Result:​
Voiceover is not announcing the "Error Message" getting while generating script.

### Expected Result:​
Voiceover should announce all the status messages including the "Error Message" getting while generating script.
### User Impact:
Voiceover user will miss the status information about the script is generating or not and if there is any error user will miss the information about that error.

### Attachment:
![generate script - satus msg for error](https://user-images.githubusercontent.com/91586101/223098388-42cc6fec-d437-4c89-b36f-9e651db179cd.gif)

**Comment by kisantia** [comment]
@manaswita-chougule I see the error being read by Voiceover:
![generateScriptBuildError](https://user-images.githubusercontent.com/31145923/223890105-04673116-1181-4e7c-96f4-79cfd1b58612.gif)

**Comment by manaswita-chougule** [comment]
@kisantia I have verified the bug in below environment:
SQL Azure Data Studio Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

The issue is still repro'ing. Please find attachment for reference. 
Could you please provide us the "SQL Azure Data Studio Version" details where the issue is not repro'ing. Thanks!

![22163_Repro](https://user-images.githubusercontent.com/91586101/224278314-f11657d5-a20c-4974-8dbe-98152a3d7f75.gif)

**Comment by kisantia** [comment]
@manaswita-chougule I'm testing in this environment:
SQL Azure Data Studio Version: 1.42.0-insider
MAC Version: macOS Monterey 12.6.3


I also tried using keyboard to navigate the dialog and voiceover is still reading the error
![errorKeyboard](https://user-images.githubusercontent.com/31145923/224418646-f4a13202-7b45-4617-8c06-3084e6d48104.gif)

**Comment by manaswita-chougule** [comment]
@kisantia As the issue is repro'ing with macOS Ventura 13.3 and not with macOS Monterey 12.6.3. could you please confirm us whether the bug is external to macOS, if yes will route it to respective team. Thanks!

**Comment by manaswita-chougule** [comment]
@Benjin Could you please provide us the attachment showing above mentioned behavior. And as the error message has few controls and (in above mentioned case) it is not reading the whole error message as well, so it is required that voiceover should announce complete message so that user will able to understand the fail reason and act accordingly. Thanks!

**Comment by Benjin** [comment]
Here you, @manaswita-chougule!  You can see Voiceover handling the contents of the error message popup at the 9 second mark.

> Voiceover should announce complete message so that user will able to understand the fail reason and act accordingly

I think it's also worth noting that the error message says to check the output pane for the details of the error, which is what Voiceover is in the middle of reading when the error pop-up occurs, and what it returns to reading for the rest of the clip (and another two minutes afterward that I trimmed).

https://user-images.githubusercontent.com/1609827/226740518-0a1d3164-ccf5-429f-af80-cb724b210753.mov

**Comment by kisantia** [comment]
@manaswita-chougule can you please verify that the error is being read by voiceover, as shown by the frame and video in Benjin's comment above?

**Comment by manaswita-chougule** [comment]
Verified the issue in below environment:
SQL Azure Data Studio Version: 1.43.0-insider (Universal)
MAC Version: macOS Ventura 13.3

The issue is fixed, hence closing the bug. Please find below attachment for reference. Thanks!

![22163_Fixed](https://user-images.githubusercontent.com/91586101/228536889-39b9ef5b-7d88-441c-8710-bf4bcea43936.gif)
